en/editor/preview/browser.md

```diff
@@ -0,0 +1,123 @@
+
+![start-scene](index/auto-refresh.jpg)
+
+## Scene of Preview
+
+The editor will use **the current scene** as start-scene of preview, we can modified it in `Project Settings-> Project Preview`.(Please refer to [Project Settings](../project/index.md) for details)
+
+![start-scene](index/start-scene.jpg)
+
+## Preview In Mobile
+
+There are the following ways to debug the preview page on the mobile phone:
+
+- **Use the mobile browser preview mode of Browser Developer Tools**
+
+- **Scan preview QR code**
+
+    Move the mouse to the ip preview address on the left side of the editor toolbar, and a preview QR code will be displayed, which can be scanned with your mobile phone.
+
+- Directly **Enter the preview address in the mobile browser**
+
+![preview-url](index/preview-url.jpg)
+
+**Notes**: Make sure the mobile phone and the computer is **on the same network segment**.Since there will be multiple networks on the computer, if the IP address of the preview URL in the editor is incorrectly, you can modify it in `Preferences-> General Settings-> Select Local Preview Address`. Please refer to the description of [preference settings](../preference/index.md) for details.
+
+## Custom Preview Template
+
+We can add custom preview template to change the preview result,just put your template in **`preview-template`** folder in the project dictionary.Editor also use template to render to `index.html`. If there is a file named `index.ejs` in this dictionary,editor will use it as the template that render to `index.html`.Your can click the menu in editor **`Project ——> Generate Preview Template`** to get the latest template use in editor.
+
+```
+project-folder
+ |--assets
+ |--build
+ |--preview-template
+        |--index.ejs
+```
+
+**Notes**: There are some preview menu items and preview debugging tools in the preview template, We recommended you keep all the content and **just add what you need** or modified code carefully to avoid some unexpected errors.. In addition, if `index.html` and `index.ejs` coexist, **`index.html` will replace `index.ejs`** as the render-template for preview page .
+
+### Examples of Use
+
+The code of this example is stored in the [test-case-3d](https://github.com/cocos-creator/test-cases-3d).
+
+1. **Get lasted preview template**
+
+     Click the menu **`Project-> Generate Preview Template`** to generate a preview template, and the address generated by the preview template will be printed in the console.
+
+2. **Add script in folder**
+
+    Add the scripts you need to use in folder, such as `test.js`, where `<%-include (cocosTemplate, {})%>` contains the default logic for starting the game.`test.js` in the example below will be loaded after the game starts.
+
+3. **Modify `index.ejs`**
+
+    Open index.ejs and modify as follows:
```

Comment:
Open index.ejs and modify as follows:
-->
Open `index.ejs` and modify it as follows: